README.md

```diff
@@ -1,16 +1,13 @@
 [![Build Status](https://dev.azure.com/Lumispy/lumispy/_apis/build/status/LumiSpy.lumispy?branchName=master)](https://dev.azure.com/Lumispy/lumispy/_build/latest?definitionId=3&branchName=master)
 ![Tests](https://github.com/lumispy/lumispy/workflows/Tests/badge.svg)
-[![Build status](https://github.com/pyxem/kikuchipy/workflows/build/badge.svg)](https://github.com/pyxem/kikuchipy/actions)
```

Comment:
While you are at it, can you update the link of the "Tests" badge? At the moment it links to the svg... which is not very useful!

Reply:
good point.